woodwork/tests/type_system/conftest.py

```diff
@@ -332,6 +332,27 @@ def pdnas(request):
     return request.getfixturevalue(request.param)
 
 
+# Empty Series Inference Fixtures
+@pytest.fixture
+def pandas_empty():
+    return [pd.Series([], dtype="object")]
```

Comment:
You can just return a Series and not a list (here and in the other fixtures), then you can remove the `for` loop in the new test you added.

Reply:
Just to clarify -- wrap L338 in a `pd.Series` call?

Reply:
No, just make the fixture look like this:
```
@pytest.fixture
def pandas_empty():
    return pd.Series([], dtype="object")
```

There is no reason to return a list that contains only a single element as far as I can tell.